fanuc_r2000ic_support/tests/roslaunch_test_r2000ic10l.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
```

Comment:
Was this file included by mistake? It appears to be a copy of `roslaunch_test_r2000ic210l.xml` below, but is called `..._r2000ic10l.xml` (note the `10` instead of `210`).